ecs/tags.go

```diff
@@ -44,15 +44,15 @@ func serviceTags(project *types.Project, service types.ServiceConfig) []tags.Tag
 	}
 }
 
-func networkTags(project *types.Project, net types.NetworkConfig) []tags.Tag {
+func networkTags(project *types.Project, name string) []tags.Tag {
 	return []tags.Tag{
 		{
 			Key:   compose.ProjectTag,
 			Value: project.Name,
 		},
 		{
 			Key:   compose.NetworkTag,
-			Value: net.Name,
```

Comment:
@ndeloof I had to adapt this to keep cloudformation_test.go unchanged. Not sure what's best, set this tag to network key, or use network name that is now prefixed with project name (cf https://github.com/docker/compose-cli/runs/1696392558#step:7:59)